data/json/terrain.json

```diff
@@ -300,6 +314,7 @@
         "type" : "terrain",
         "id" : "t_dirt",
         "name": "dirt",
+        "description": "It's dirt. Looks like some fine soil for tillage, or dug out for construction projects.",
```

Comment:
As mentioned in the previous PR, the second sentence reads a bit funny.

> Looks like some fine soil for tillage, or dug out for construction projects.

First, "tillage" should be "tilling," the verb. "Tillage" is the method/system of tilling.  Secondly, the two clauses rely on the same predicate ("looks like some fine soil"), and it's causing some confusion. The second clause is read as:

> Looks like some fine soil for dug out for construction projects.

Here are some alternatives:

> It's dirt. The fine soil could be tilled for crops or dug out for construction projects.

> It's dirt. Looks like some fine soil for tilling. Could also be dug out for construction projects.

> It's dirt, ideal for tilling. Could also be dug out for construction projects.

Reply:
Correct, tillage is the noun for 
> the preparation of land for growing crops.

So to illustrate its usefulness here: "Looks like some fine soil for hydroponics" - it wouldn't be "Looks like some fine soil for hydroponically". So to correct for "tilling", it would be stated as "Tilling this fine soil would be useful ... ". So, "tillage" is the correct form here.

The second part could be clunky, depending on the reader.